app/src/main/java/org/amfoss/templeapp/activities/DeleteData.java

```diff
@@ -43,13 +43,13 @@ public void radioGroupOnCheckedChanged(RadioGroup group, int checkedId) {
                 case R.id.radio_donate:
                     binding.totalView.setVisibility(View.VISIBLE);
                     flag = 0;
-                    Toast.makeText(getBaseContext(), getString(R.string.delete_donated), Toast.LENGTH_LONG)
+                    Toast.makeText(getBaseContext(), getString(R.string.delete_donated), Toast.LENGTH_SHORT)
                             .show();
                     break;
                 case R.id.radio_pooja:
                     binding.totalView.setVisibility(View.VISIBLE);
                     flag = 1;
-                    Toast.makeText(getBaseContext(), getString(R.string.delete_pooja), Toast.LENGTH_LONG)
+                    Toast.makeText(getBaseContext(), getString(R.string.delete_pooja), Toast.LENGTH_SHORT)
```

Comment:
Change to `getBaseContext` instead of `getApplicationContext` as this is that preferred way since this Context lives untill Application shuts down.
I am seeing so many unused changes please change that

Reply:
@Chromicle 
Can you please highlight the unused changes . 
So that i can know the specifics.

Reply:
Should i also change to getBaseContext instead of getApplicationContext in MainActivity ?